src/components/Input/DefaultInput.tsx

```diff
@@ -0,0 +1,124 @@
+import type {
+  InputProps as DefaultInputProps,
+  InputStyleOption,
+  InputStylesheet
+} from './index'
+import type { ReactElement } from 'react'
+import styled from '@emotion/styled'
+import type { StyledProps } from '@types'
+
+type DefaultSTyleOption = 'FONT' | 'HEIGHT' | 'PADDING_BOTTOM'
+type DefaultStylesheet = InputStyleOption<DefaultSTyleOption>
+
+interface StyledPriceUnitProps {
+  inputStylesheet: DefaultStylesheet
+}
+type StyledInputProps = StyledProps<DefaultInputProps, 'isPrice'> &
+  StyledPriceUnitProps
+type StyledStatusProps = StyledProps<DefaultInputProps, 'status'>
+
+const DEFUALT_STYLESHEET: InputStylesheet<DefaultStylesheet> = {
+  large: {
+    FONT: 'subtitle01M',
+    HEIGHT: 54,
+    PADDING_BOTTOM: 16
+  },
+  small: {
+    FONT: 'body02M',
+    HEIGHT: 40,
+    PADDING_BOTTOM: 10
+  }
+} as const
+
+export const DefaultInput = ({
+  label,
+  status = 'default',
+  message = '',
+  isPrice = false,
+  inputSize = 'large',
+  ...args
+}: DefaultInputProps): ReactElement => {
+  const hasMessage = status !== 'none'
+  const inputStylesheet = DEFUALT_STYLESHEET[inputSize]
+
+  return (
+    <StyledWrapper>
+      <StyledLabel>
+        {label}
+        <StyledInput
+          inputStylesheet={inputStylesheet}
+          isPrice={isPrice}
+          {...args}
```

Comment:
```suggestion
          isPrice={isPrice}
          type={isPrice ? 'number' : 'text'}
          {...args}
```
`isPrice` props로 type까지 제어되면 좋을 것 같습니다!

Reply:
[요기](dc4e0b948281b0d69dfbda51ee92ad4c4dbaad47)에 반영 완료 ! ☺️
number로 사용되는 경우 값에 컴마를 찍게 되는 경우가 있어서 타입을 number로 지정해주지 않는 방향으로 수정했습니다
number로 지정해주면 컴마가 제거되더라구융!